server/src/instant/reactive/session.clj

```diff
@@ -542,6 +542,10 @@
       (let [pending-handlers (:pending-handlers (:session/socket session))
             in-progress-stmts (sql/make-statement-tracker)
             debug-info (atom nil)
+            app-id (-> session :session/auth :app :id)
+            timeout-ms (or (when app-id
```

Comment:
Could we also add the timeout-ms to the `:attributes` for `handle-receive`? Might be useful for debugging later.